SayTheirNames/Source/Controller/More/MoreView.swift

```diff
@@ -51,26 +110,26 @@ final class MoreView: UIView {
     
     /// Configures constraints for subviews
     private func setupSubviews() {
-        let contentView = UIStackView(
-            arrangedSubviews:
-            [moreCard,
-             historySection(),
-             developerSection(),
-             designerSection(),
-             thankYouLabel])
-        
-        contentView.axis = .vertical
-        contentView.spacing = Theme.Components.Padding.extraLarge
-        contentView.translatesAutoresizingMaskIntoConstraints = false
+        let stack = UIStackView()
```

Comment:
not something introduced in this PR, but the screen needs some spacing below this stack view: 
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 17 16 29](https://user-images.githubusercontent.com/11408899/84404581-52ac6600-ac07-11ea-91b1-86d8cefd6423.png)